DOC-3394-added a note about Values YAML (#2629)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/environments/service-overrides.md

When you have a values YAML file at two or more of the environment **Service Overrides**, **Environment Configuration**, and the service itself, Harness merges the files into a single values YAML for deployment. This merging is performed at pipeline execution runtime.

:::info limitation

> You cannot check the parametrized values in the values YAML. However, you can view these values in the pipeline execution console view for any deployment type under the Initialize section if you're referencing the values YAML in the corresponding manifest.

Reviewer: Please consider if this works:

....However, you can view these values in the pipeline execution console view for any deployment type under the Initialize section, when referencing the values YAML in the corresponding manifest.

> You cannot check the parametrized values in the values YAML. However, you can view these values in the pipeline execution console view for any deployment type under the Initialize section if you're referencing the values YAML in the corresponding manifest.

Author: ```suggestion
You cannot check the parametrized values in the values YAML. However, you can view these values in the pipeline execution console view for any deployment type under the Initialize section, when referencing the values YAML in the corresponding manifest.
```
